### PR TITLE
Add other Snuba related config changes

### PR DESCRIPTION
**Pull request by mattrobenolt**
This is ignoring TSDB changes so far.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,10 @@ A service providing fast event searching, filtering and aggregation on arbitrary
 
 ## Sentry + Snuba
 
-Add the following line to `~/.sentry/sentry.conf.py`:
+Add/change the following lines to `~/.sentry/sentry.conf.py`:
```
Comment (tkaemming) [inline README.md]: "to" should probably be "in" here

Comment (mattrobenolt) [reply README.md]: I can always count on Ted. Reliable Ted as he's called.

Comment (tkaemming) [reply README.md]: Actually, I'm not sure what is more grammatically correct here on second thought — it'd be "add to" or "change in" but I'm not sure which takes priority when written like this? If it was written as "Add or change the the following lines", I'd think "in" would be the correct one here?

<big>🤔</big>

Comment (mattrobenolt) [reply README.md]: image

Comment (JTCunning) [reply README.md]: screenshot 2019-02-28 13 17 24
We're not done here. The people need to know what we choose.

**Pull request merged**